framework/assets/yii.js

```diff
@@ -144,8 +144,8 @@ yii = (function ($) {
          * @param $e the jQuery representation of the element
          */
         handleAction: function ($e, event) {
-            var method = $e.data('method'),
-                $form = $e.closest('form'),
+            var $form = $e.attr('data-form') ? $('#' + $e.attr('data-form')) : $e.closest('form'),
```

Comment:
Yes: https://jsfiddle.net/gw6uvoLw/5/
Basically if you set it via something else than jQuery it is not the same. The example shows how, for example, css, does not care about `$.data` since it is limited to jQuery.

The same holds for pure JS.